src/Amplitude.js

```diff
@@ -254,6 +258,24 @@
             return false;
         }
 
+        function convertArrayAttrs(customAttributes) {
+            for (var key in customAttributes) {
+                if (typeof customAttributes[key] === 'string' && forwarderSettings.sendEventAttributesAsObjects === 'True') {
```

Comment:
you could prob just do a if (forwarderSettings.sendEventAttributesAsObjects) return customAttributes untouched at the top vs checking in the loop.